plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -55,6 +55,7 @@
 #include <mbedtls/ctr_drbg.h>
 #include <mbedtls/md5.h>
 #include <mbedtls/base64.h>
+#include <mbedtls/x509_crt.h>
```

Comment:
This was added in support for newer mbedtls, but does it exist in older mbedtls? Does it need to be wrapped in an ifdef?

Reply:
It exists in 2.11 (current dependencies.zip) too, I'm not actually sure if it's needed. My MSVC just kept complaining about something_x509_crt_t not being defined, so I added it.